activation/atxdb.go

```diff
@@ -20,34 +20,58 @@ import (
 	"github.com/spacemeshos/post/shared"
 )
 
-const topAtxKey = "topAtxKey"
+const (
+	namespaceAtx       = "a"
+	namespaceEpoch     = "e"
+	namespaceTop       = "p"
+	namespaceHeader    = "h"
+	namespaceTimestamp = "t"
+)
 
 func getNodeAtxKey(nodePK string, targetEpoch types.EpochID) []byte {
```

Comment:
what do you think about putting all the leveldb keys functions and prefixes in one file? for me it's better readability and potentially easier to prevent collision.

Reply:
do you mean like for all databases? because we are using multiple isolated databases in current code

Reply:
yes. i know we are using multiple isolated databases in current code. but it would be nice to hide those details somewhere else instead of mixing with application logic. but your call.

Reply:
i think it makes more sense to do it when we will be merging them